src/Command/Setup.php

```diff
@@ -81,7 +84,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function validateVar($value)
     {
         if (!preg_match('/^([-A-Z0-9])*$/i', $value)) {
-            throw new RuntimeException(sprintf('This value must contain only alphanumeric characters and hyphens.', $value));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
The idea behind this syntax is that `sprintf` is not that much important and can stay inlined with the exception constructor call, whereas listing the exception message along with the arguments is visually more effective in order to apprehend the exception message placeholders.